### ISSUE TITLE
presentation-role-conflict for info icon on  action panel 

### ISSUE THREAD

**Issue by Xiaoyu-Huang** [Azure/LogicAppsUX#4697]
### Describe the issue

Run Accessibility insights extension fast pass, it detected issues on info icon on action panel. Please see image below.

![image](https://github.com/Azure/LogicAppsUX/assets/22843713/0eabe814-a9fe-4a99-b4c1-70221b5be57a)

here is the failure details:
Title: ,WCAG 1.3.1,Best Practice: Elements marked as presentational should not have global ARIA or tabindex to ensure all screen readers ignore them (.msla-info-dot[alt=""])
Tags: Accessibility, , WCAG 1.3.1, Best Practice, presentation-role-conflict

Issue: Elements marked as presentational should not have global ARIA or tabindex to ensure all screen readers ignore them (presentation-role-conflict - https://dequeuniversity.com/rules/axe/4.7/presentation-role-conflict?application=msftAI)

Target application: IcM - https://portal.microsofticm.com/imp/v5/automation/workflows/create

Element path: div[data-list-index="0"] > .mlsa-browse-list-tile-wrapper > .msla-browse-list-tile > .msla-connector-summary-card > .msla-connector-summary-header > .root-209.ms-TooltipHost[role="none"] > .msla-info-dot[alt=""]

Snippet: <img class="msla-info-dot" alt="" src="data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 11 11'%3e %3cdefs%3e %3cstyle%3e .a %7b fill: %23919191%3b %7d %3c/style%3e %3c/defs%3e %3ctitle%3eInfoTiny%3c/title%3e %3cpath class='a' d='M12105.5%2c5893a5.45%2c5.45%2c0%2c0%2c1-1.46-.19%2c5.49%2c5.49%2c0%2c0%2c1-3.84-3.84%2c5.6%2c5.6%2c0%2c0%2c1%2c0-2.93%2c5.49%2c5.49%2c0%2c0%2c1%2c3.84-3.84%2c5.6%2c5.6%2c0%2c0%2c1%2c2.93%2c0%2c5.49%2c5.49%2c0%2c0%2c1%2c3.84%2c3.84%2c5.6%2c5.6%2c0%2c0%2c1%2c0%2c2.93%2c5.49%2c5.49%2c0%2c0%2c1-3.84%2c3.84A5.45%2c5.45%2c0%2c0%2c1%2c12105.5%2c5893Zm0-10.27a4.63%2c4.63%2c0%2c0%2c0-1.27.17%2c4.88%2c4.88%2c0%2c0%2c0-1.14.48%2c4.77%2c4.77%2c0%2c0%2c0-1.71%2c1.71%2c4.87%2c4.87%2c0%2c0%2c0-.48%2c1.14%2c4.73%2c4.73%2c0%2c0%2c0%2c0%2c2.53%2c4.87%2c4.87%2c0%2c0%2c0%2c.48%2c1.14%2c4.77%2c4.77%2c0%2c0%2c0%2c1.71%2c1.71%2c4.88%2c4.88%2c0%2c0%2c0%2c1.14.48%2c4.74%2c4.74%2c0%2c0%2c0%2c2.53%2c0%2c4.87%2c4.87%2c0%2c0%2c0%2c1.14-.48%2c4.77%2c4.77%2c0%2c0%2c0%2c1.71-1.71%2c4.88%2c4.88%2c0%2c0%2c0%2c.48-1.14%2c4.74%2c4.74%2c0%2c0%2c0%2c0-2.53%2c4.87%2c4.87%2c0%2c0%2c0-.48-1.14%2c4.77%2c4.77%2c0%2c0%2c0-1.71-1.71%2c4.87%2c4.87%2c0%2c0%2c0-1.14-.48A4.64%2c4.64%2c0%2c0%2c0%2c12105.5%2c5882.73Zm-0.5%2c2.27h1v0.8h-1V5885Zm0%2c2h1v3h-1v-3Z' transform='translate(-12100 -5882)' /%3e%3c/svg%3e" tabindex="0">

How to fix: 
Fix all of the following:
  Element is not focusable.

Environment: Microsoft Edge version 124.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.41.0 (axe-core 4.7.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

### Are you using new designer or old designer

New Designer (With action panels)

### Browser

Browser: Edge

logic app library version:

    "@microsoft/logic-apps-designer": "4.1.1",
    "@microsoft/logic-apps-shared": "4.1.1"
